application/views/js_init/phonebook/js_phonebook.php

```diff
@@ -45,11 +45,13 @@
 											buttons: {
 												"Okay": function() {
 													$(this).dialog("close");
+													location.reload();
```

Comment:
A reload of page on button close isn't the best experience. We should instead trigger the ajax reload of messages if possible (don't recall how its implement in other cases) .

Reply:
Ok, I have no interest in learning ajax. Maybe you can do it now that you know where the issue lies then. Thanks.

Reply:
Just saw that when adding contacts by csv, the page is also reloaded in code with load->view...

Reply:
Hmm, looks like all pages in kalkun is made without any aync/ajax calls and are rendered from PHP only.

I think   `location.reload();` here is not required. the one at line 54 is enough.

Reply:
I guess both are needed, one if the user clicks close, one if the user doesn't click but the dialog auto closes after timeout. Am I wrong?